### PR TITLE
clean up readme, add port on run

### PR DESCRIPTION
**Pull request by jobin212**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,40 +10,49 @@ A place for [Recursers](https://www.recurse.com) to color pixels, inspired by
...
-- Run rc-place app
-```bash
-make run
-```
+Create an OAuth application at [https://www.recurse.com/settings/apps](https://www.recurse.com/settings/apps) with proper redirect URI (http://localhost:8080/auth for local run). 
+**Make sure to set your app's ID, Secret, and Redirect URI in your environmental variables** (see [.env.example](.env.example)). You can optionally set your own redis host and password.
+
+```shell
+# Load your environmental variables after setting them
+🎨 source .env.example
```
Comment (mcastorina) [inline README.md:18]: What's with the palette?

File: README.md [hunk 2]
```diff
@@ -10,40 +10,49 @@ A place for [Recursers](https://www.recurse.com) to color pixels, inspired by
...
 
 ## TODO
 - Follow up on other architecture decisions detailed in [How We Built r/place](https://www.redditinc.com/blog/how-we-built-rplace), e.g. Redis
 - Add API to load entire board state, update frontend to follow
 - Make board big (1000 x 1000) and full screen
-- Look into performance we could get on frontend or backend
 - Add user information to tiles -- who placed what tile
-- Bot example
-- Refactor main.go, get a good go score
-- Add github actions, make it so @miccah can deploy
```
Comment (mcastorina) [inline README.md:42]: I have confirmed that I can deploy

**Comment by jeffawang** [comment]
LGTM

File: main.go [hunk 3]
```diff
@@ -54,6 +54,7 @@ func main() {
 		}
 		serveWs(hub, &session.User, w, r)
 	})
+	log.Printf("Running on port %s\n", *addr)
```
Comment (mcastorina) [inline main.go:57]: I think this should be printed after the server starts?

**Review by jeffawang — APPROVED** [review]

File: main.go (hunk 3 above)
Comment (jobin212) [reply main.go:57]: I agree! But it looks like the way http.ListenAndServe() runs, that code is never reached unless we have an err :/

File: README.md (hunk 1 above)
Comment (jobin212) [reply README.md:18]: We're artists now :)

**Review by mcastorina — APPROVED** [review]
lgtm

**Pull request merged**